echopype/visualize/api.py

```diff
@@ -33,13 +33,13 @@ def create_echogram(
     range_kwargs : dict
         Keyword arguments dictionary for computing range.
         Keys are `env_params`, `waveform_mode`, and `encode_mode`.
-    water_level : int, float, xr.DataArray, or bool, optional
-        Water level data array for platform water level correction.
-        Note that auto addition of water level can be performed
+    range_offset : int, float, xr.DataArray, or bool, optional
+        Range offset data array for platform range offset correction.
+        Note that auto addition of range offset can be performed
         when data is an EchoData object by setting this argument
-        to `True`. Currently because the water level information
+        to `True`. Currently because the range offset information
```

Comment:
The sentence "Currently because the range offset information is not available ..." will need to be updated, since this PR is adding the functionality to add it to the Sv dataset when available in the converted file.